src/main/java/org/jfrog/hudson/pipeline/common/types/buildInfo/BuildInfoAccessor.java

```diff
@@ -123,7 +123,9 @@ public BuildInfoDeployer createDeployer(Run build, TaskListener listener, Artifa
     }
 
     public void appendVcs(Vcs vcs) {
-        this.buildInfo.appendVcs(vcs);
+        if (StringUtils.isNotEmpty(vcs.getRevision()) && StringUtils.isNotEmpty(vcs.getUrl())) {
```

Comment:
What is the scenario that forces us to do this?
My concern is that this may hide a real bug that the code may have.